# Dead Reckoning node configuration for ROS2
dead_reckoning_node:
  ros__parameters:
    # Localization node
    dvl_max_velocity: 2.0

    # [x y z roll pitch yaw] in radians
    imu_pose: [0.0, 0.0, 0.0, -1.5708, 0.0, 0.0]  # roll = -90 degrees
    # imu_pose: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]

    # keyframe spacing for the dead reckoning system
    keyframe_duration: 1.0
    keyframe_translation: 4.0
    keyframe_rotation: 0.5236  # 30 degrees in radians
